viadot/tasks/azure_sql.py

```diff
@@ -274,7 +279,23 @@ def run(
         azure_sql = AzureSQL(credentials=credentials)
 
         # run the query and fetch the results if it's a select
-        result = azure_sql.run(query)
+        if if_failed == "break":
+            result = azure_sql.run(query)
+        elif if_failed == "skip":
+            count = 0
+            for char in enumerate(query):
+                if char[1] == ";":
```

Comment:
It is error-prone. If the user doesn't end the line with a semicolon, it will not work properly.